cpp/velox/tests/VeloxToSubstraitTypeTest.cc

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Question: Hmm, should License header add Intel's as well since moving under Intel's repo?

Reply:
Thanks for pointing out. Fixed.